Description: Typo fix
Author: Lee Garrett <lgarrett@rocketjump.eu>
Bug-Debian: https://bugs.debian.org/426621
Forwarded: no
Reviewed-By: Andreas Barth <aba@not.so.argh.org>
Last-Update: 2016-06-21

Index: rp-pppoe/doc/HOW-TO-CONNECT
===================================================================
--- rp-pppoe.orig/doc/HOW-TO-CONNECT
+++ rp-pppoe/doc/HOW-TO-CONNECT
@@ -27,7 +27,7 @@ should display something like this:
 
 	eth0      Link encap:Ethernet  HWaddr 00:60:67:62:31:D4
 
-plust some more lines.  Your HWaddr will be different.  As long as you see
+plus some more lines.  Your HWaddr will be different.  As long as you see
 the HWaddr line, your card should be working.
 
 DO NOT assign an IP address to the Ethernet card.  DO NOT configure the
